src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -0,0 +1,41 @@
+(ns org.broadinstitute.firecloud-ui.common.style
+  (:require [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+(def colors {:link-blue "#6690c5"
+             :line-gray "#e6e6e6"
+             :button-blue "#457fd2"
+             :background-gray "#f4f4f4"
+             :text-light "#7f7f7f"
+             :text-gray "#666"})
+
+
+(defn form-label [text]
```

Comment:
My idea was to visually match other types of elements which are just nouns.  I could make the style maps private.